### PR TITLE
update names for msbuild sdk style projects

### PR DESCRIPTION
**Pull request by kisantia**
Updates the file names to be a little more clear so they mention "msbuild sdk style" instead of just "new style." The first msbuild sdk style test projects had new in the name because they were for the new project scenario, but then I accidentally kept the new in the names for the open project scenarios that got added later.

### CODE REVIEW

**Comment by zijchen** [comment]
I'm thinking ".NET SDK-style project" and not mention "msbuild" at all, since with the SDK we support "dotnet" commands

**Comment by kisantia** [comment]
It's a little confusing since the [docs](https://docs.microsoft.com/en-us/visualstudio/msbuild/how-to-use-project-sdk?view=vs-2022) are say msbuild project sdk, then refer to it as "project sdk."  I removed the msbuild and just kept the "SdkStyle" in the names since the sdk attribute is what we're looking for in the sqlproj.

**Comment by zijchen** [comment]
I think that's good, from the main doc for [.NET project SDKs](https://docs.microsoft.com/en-us/dotnet/core/project-sdk/overview), it mentions
> A project that references a project SDK is sometimes referred to as an SDK-style project.

**Review by zijchen — APPROVED** [review]

File: extensions/sql-database-projects/src/models/project.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export class Project implements ISqlProject {
 	private _preDeployScripts: FileProjectEntry[] = [];
 	private _postDeployScripts: FileProjectEntry[] = [];
 	private _noneDeployScripts: FileProjectEntry[] = [];
-	private _isMsbuildSdkStyleProject: boolean = false;
+	private _isSdkStyleProject: boolean = false;
```
Comment (Benjin) [inline extensions/sql-database-projects/src/models/project.ts:37]: Might be worth adding a comment here with a link to https://docs.microsoft.com/en-us/dotnet/core/project-sdk/overview so anyone reading the code can understand exactly what this means

**Review by Benjin — APPROVED** [review]

**Pull request merged**